src/util/WebUtils.js

```diff
@@ -58,3 +57,44 @@ export const stringifyUriAndQuery = (uri: string, query: UriQueryMap): string =>
 export const omitLastChar = (uri: string, lastChar: string): string => {
   return uri.substr(-1) === lastChar ? uri.substr(0, uri.length - 1) : uri
 }
+
+/**
+ * -Replaces reserved characters with escape sequences representing the UTF-8
+ * encoding of the character.
+ * -Joins each query with '&'
+ */
+export const stringifyQuery = (query: UriQueryMap): string => {
+  return Object.keys(query)
```

Comment:
should probably use `url-parse` which does full work of parsing and encoding urls.

Reply:
Added TODO